app/src/main/java/org/jellyfin/mobile/MainActivity.kt

```diff
@@ -190,6 +190,7 @@ class MainActivity : AppCompatActivity() {
                 fragment.onUserLeaveHint()
             }
         }
+        super.onUserLeaveHint()
```

Comment:
This was added due to the linter/editor complaining it should not have been left out.